LICENSE

```diff
@@ -1,6 +1,7 @@
-MIT License
 
-Copyright (c) 2022 Thundra
+The MIT License (MIT)
+
+Copyright (c) 2018 GitHub, Inc. and contributors
```

Comment:
Why we changed copyright to Github?

Reply:
It has been changed after template initialized.